MacMagazine/Supporting Files/Info.plist

```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>3.0.1</string>
+	<string>3.0.2</string>
```

Comment:
Pode manter na versão `3.0.1`, quando for fechar o release pra App Store eu vou atualizar isso conforme as novidades (nesse caso, possivelmente será `3.1`)